### DISCUSSION TITLE
View releases in the app

### DISCUSSION THREAD

**Discussion by fridtjof** [community/community#3032] [Mobile] [answered]
Right now, when I have a link to a repository where I want to look at the releases page, under certain conditions the app will open the link for me. That usually takes some time for the app to start, and then load the repository for display. To get to the releases page from there, I have to tap the Share button, "Open in Safari" and then go to the releases section after the page finishes loading (again, kind of). It would be so much easier to just be able to view the releases section in the Github app. Many projects put their release notes in there exclusively, and currently my choices are either enduring the annoying sidestep to the app, or uninstalling it altogether.

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback. Supporting "Releases" is something already on our radar. Will keep you posted as we get any updates!

**Reply by macintacos** [reply]
Hey @candyho! It looks like the iOS app supports GitHub releases now, however if you click on a release from "somewhere else" (like Twitter for example) there's no easy way to that repo's main page. Any chance that could be addressed?

**Reply by candyho** [reply]
Hey thanks for reporting! We are aware of this and tracking this. Hopefully we could land on a solution soon. And YES! Releases is available on both iOS and Android. You can view your releases on mobile now.
